Core/fingerprint.js

```diff
@@ -180,11 +180,26 @@
     var excludeTempStorage = false;
     var excludeBattery = false;
     var excludeScreenSize = false;
+    var excludeAll = false;
     var domainParts = topLevelUrl && topLevelUrl.host ? topLevelUrl.host.split(".") : [];
 
+    const userExcluded = `
+                    ${userUnprotectedDomains}
+                    `.split("\n").filter(domain => domain.trim() == topLevelUrl.host).length > 0;
+    if (userExcluded) {
+        return;
+    }
+    
     // walk up the domain to see if it's unprotected
     while (domainParts && domainParts.length > 1) {
         let partialDomain = domainParts.join(".")
+        
+        excludeAll = `
+                ${tempUnprotectedDomains}
```

Comment:
In the ESLint change for iOS we agreed to make this to `$TEMP_UNPROTECTED_DOMAINS$` and all the other corresponding ones the same format too.

Reply:
Yes that _just_ landed so I'll be having to update these anyway.

Reply:
Course, it was a FYI not error report :)